les/coinbase-transaction-v1.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+   "@context": {
+      "@version": 1.1,
+      "cbt": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#",
+      "schema": "http://schema.org/",
+      "CoinbaseTransaction": {
+         "@id": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#rwc",
+         "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "transactionId": "schema:String",
```

Comment:
Since this is already about a transaction alone, I think we can leave out the transaction Prefixes.

Reply:
Oh, hmzz, we probably can remove the first id=@id, given the txIds will be unique anyway